code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -163,4 +163,14 @@
 
 /obj/effect/decal/cleanable/soot/New()
 	..()
-	dir = pick(cardinal)
\ No newline at end of file
+	dir = pick(cardinal)
+
+/obj/effect/decal/lspaceclutter

```

Comment:
No specific clutter object beyond the 'generic' one, but I just noticed I forgot to add the 'cleanable' thing to it so lemme fix that real quick